packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -57,12 +57,20 @@ function printAttributes(attributes, print, indent, colors, opts) {
 }
 
 const print = (
-  element: any,
+  element: HTMLElement | Text | Comment,
   print: Print,
   indent: Indent,
   opts: Options,
   colors: Colors,
-) => {
+): string => {
+  if (element instanceof Text) {
```

Comment:
It won't work outside of having the jsdom globals added as globals in node. Assuming that's nok OK, any idea on how to make flow accept that `element.nodeType === 3` is the "same" as `element instanceof Text`?